appveyor.yml

```diff
@@ -112,7 +112,7 @@ build_script:
           }
         }
       }
-  - if "%RUN_GCLIENT_SYNC%"=="true" ( gclient sync --with_branch_heads --with_tags --ignore_locks)
+  - if "%RUN_GCLIENT_SYNC%"=="true" ( gclient sync --with_branch_heads --with_tags)
```

Comment:
🤔  I think we can get rid of `with_branch_heads` and `with_tags` too now 🤔  will open a separate PR.